lib/msal-browser/test/interaction_client/BaseInteractionClient.spec.ts

```diff
@@ -117,11 +154,34 @@ describe("BaseInteractionClient", () => {
             // @ts-ignore
             pca.browserStorage.setAccount(testAccount1);
             // @ts-ignore
+            pca.browserStorage.setIdTokenCredential(idToken1);
+            // @ts-ignore
             pca.browserStorage.setAccount(testAccount2);
+            // @ts-ignore
+            pca.browserStorage.setIdTokenCredential(idToken2);
+
+            jest.spyOn(
+                CacheManager.prototype,
+                "getAuthorityMetadataByAlias"
+            ).mockImplementation((host: string) => {
+                const metadata =
+                    DEFAULT_TENANT_DISCOVERY_RESPONSE.body.metadata[0];
+                const openIdConfigResponse =
+                    DEFAULT_OPENID_CONFIG_RESPONSE.body;
+                const authorityMetadata = new AuthorityMetadataEntity();
+                authorityMetadata.updateCloudDiscoveryMetadata(metadata, true);
+                authorityMetadata.updateEndpointMetadata(
+                    // @ts-ignore
+                    openIdConfigResponse,
+                    true
+                );
+                return authorityMetadata;
+            });
         });
 
         afterEach(() => {
             window.sessionStorage.clear();
+            jest.clearAllMocks();
```

Comment:
Clear only resets counters and such, restore will reset the function to its original state
```suggestion
            jest.restoreAllMocks();
```

Reply:
Didn't know that, thanks!